freqtrade/rpc/api_server/api_ws.py

```diff
@@ -115,26 +115,34 @@ async def message_endpoint(
                     # Process the request here
                     await _process_consumer_request(request, channel, rpc)
 
-            except WebSocketDisconnect:
+            except (
+                WebSocketDisconnect,
+                websockets.exceptions.WebSocketException
+            ):
```

Comment:
If you import the `WebSocketException` directly, it'll make this line format nicer.
we don't use websockets elsewhere in this file - so it would make this part a bit nicer to look at / read